python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -42,6 +43,7 @@ def __new__(
         container_name: str,
         command: Optional[Sequence[str]],
         log_configuration: Optional[Mapping[str, Any]],
+        repository_credentials: Optional[str],
```

Comment:
```suggestion
        repository_credentials: Optional[str]=None,
```

Also, you might want to rearrange the order (maybe sticking it at the bottom?) throughout the call stack.

What's happening here is that named tuple's parameter is being type annotated as an optional string, but no default is ever set.

```
>>> from typing import Optional
>>> def foo(bar: Optional[str]):
...     return bar
... 
>>> foo()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() missing 1 required positional argument: 'bar'
>>> def foo(bar: Optional[str] = None):
...     return bar
... 
>>> foo()
>>> 
```